### PR TITLE
GEOPY-855

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-855 - **

### CODE REVIEW

**Comment by andrewg-mira** [comment]
This runs without error locally? Hmmmm... I'm going to try to reproduce it.

File: omf/fileio/geoh5.py [hunk 1]
```diff
@@ -261,6 +261,9 @@ def from_omf(self, element, **kwargs) -> Data | dict:
             else:
                 kwargs["association"] = "VERTEX"
 
+            # if isinstance(parent, BlockModel) and "values" in kwargs:
+            #     kwargs["values"] = kwargs["values"].reshape(parent.shape)[:, :, ::-1].flatten()
```
Comment (benk-mira) [inline omf/fileio/geoh5.py:265]: cleanup

File: omf/fileio/geoh5.py [hunk 2]
```diff
@@ -802,43 +805,49 @@ def collect_omf_attributes(cls, element, **kwargs) -> dict:
...
-                np.arctan2(element.geometry.axis_v[1], element.geometry.axis_v[0])
-            )
-        ) % 360
-
-        if azimuth != 0:
-            kwargs.update({"rotation": azimuth})
-
-        kwargs.update({"origin": np.r_[element.geometry.origin]})
+        offsets = np.c_[offsets].sum(axis=1)
+        kwargs["z_cell_delimiters"] = (
```
Comment (benk-mira) [inline omf/fileio/geoh5.py:816]: `kwargs["z_cell_delimiters"] *= element.geometry.axis_w[-1]`

**Review by benk-mira — CHANGES_REQUESTED** [review]
Looks good, couple comments

**Review by benk-mira — APPROVED** [review]
perfect!

**Pull request merged**